src/app/shared/components/managecitations/managecitations.component.html

```diff
@@ -50,87 +50,148 @@
 			<table class="custom-table table-striped table-scroll mtop-sm no-scrollbars">
 				<thead>
 					<tr>
-						<th class=citationHeader>Citation</th>
-						<th class=citationEditHeader></th>
-						<th class=regRegionHeader>Associated Regression Regions</th>
-						<th class="empty-cell regRegionEditHeader">
+						<th *ngIf="!inGagePage" class=citationHeader>Citation</th>
+						<th *ngIf="!inGagePage" class=citationEditHeader></th>
+						<th *ngIf="inGagePage" class="width-5">ID</th>
+						<th *ngIf="inGagePage" class="width-85">Citation</th>
+						<th *ngIf="inGagePage" class="width-10"></th>
```

Comment:
It looks like some of these are repeats, i.e. the citationHeader and citationEditHeader. Those two should be in there regardless. You can conditionally set the class if you need to, but otherwise just adding the ID here and keeping the others should be fine.

Reply:
Yea, so the reason why there are so many repeats here and especially the repeated columns below is because there needs to be different styling depending on whether the Citation Modal is being opened in a GagePage or from the MainView. Is there a way to make the CSS class dependent upon a Boolean variable? For instance; ngIf="inGagePage=true" class = A, else class = B.

Reply:
Yes, you can conditionally set the class by doing something like `[class.width-5]="inGagePage"`

Reply:
Another thought is you could set a conditional class higher up, something like `gageStatsTable`, then use the existing classes here (citationHeader, etc.) combined with that class in the scss file, e.g. `.gageStatsTable.citationHeader` (it might be different for scss v. css)